cli/test/fixtures/dobetterweb/dbw_tester.js

```diff
@@ -8,10 +8,7 @@
 
 (function() {
 
-const params = new URLSearchParams(location.search);
-
-if (location.search === '' || params.has('deprecations')) {
-  window.webkitStorageInfo.PERSISTENT; // FAIL(deprecations)
```

Comment:
Drive-by: `webkitStorageInfo` was removed (no longer just a deprecation) which was causing an extra error to appear in `errors-in-console`. We had a note to remove when M110 hit stable as well.